### ISSUE TITLE
[QA] Sound Settings Not Affecting Scene Audio

### ISSUE THREAD

**Issue by Ludmilafantaniella** [decentraland/unity-explorer#2294]
### **Explorer Alpha build versions:**

- **v0.29.0**
- **v0.76.0**
- **v0.90.0**

### **Issue Description:**
In certain scenes with their own audio, such as Casa Roustan (37,-114) or Doll House (-102,-97), the in-game sound settings do not control the scene's audio. The volume should be adjustable through the sound settings but currently remains unaffected.

📌 Notes
- The issue affects scenes with their own audio implementation, independent of global sound settings.

📝 LOGS:
[Player.log](https://github.com/user-attachments/files/20796422/Player.log)
[Player-prev.log](https://github.com/user-attachments/files/20796421/Player-prev.log)

### **STR:** 
1. Launch the explorer.
2. Go to Casa Roustan (37,-114) or Doll House (-102,-97).
3. Open the settings menu and adjust the sound settings to lower the volume.
4. Observe the audio from the scene.

⚠ UPDATED STR:
1. Enter the Game Night Arena scene. (-8,88)
2. Walk from Game Night Arena into the Game Night General scene.
3. While in the General scene, adjust the volume using the in-scene settings (e.g., lower or mute it).
4. Walk back and forth between the Arena and the General scene.
5. Attempt to adjust the volume again from the settings panel in the General scene.

### ✅ Expected Result
The sound settings should adjust the audio for all in-scene sounds, allowing the user to lower or mute it.

#### ❌ Actual Result
The scene's audio remains unaffected by the sound settings, and the volume does not decrease.

https://github.com/user-attachments/assets/c8f31199-a80e-4ba7-9840-1b966dcdcd57

### ➗ **Repro Index:**
5/10

#### **Additional Notes:**
Windows 11. 12th Gen Intel(R) Core(TM) i7-12650H 2.30 GHz. nvidia geforce RTX
Macbook M1 Pro Sonoma 14.5

**Comment by lorenzo-ranciaffi** [comment]
I noticed that on Windows the audio coming from the scene is routed through the "World" channel but on Mac is not, looks like it comes from another source

**Comment by fcolarich** [comment]
@davidejensen 
Basically all starts here -> `AudioSourcesPlugin`.cs
Seems like as Lorenzo said, the audioMixerGroup is not being assigned correctly.
This value is obtained from the asset audioMixer stored in the settings of the Plugin.
Then in `UpdateAudioSourceSystem `this happens                
`audioSourceComponent.SetAudioSource(audioSourcesPool.Get()!, (worldGroup is { Length: > 0 } ? worldGroup[0] : null)!);`
which should assign the correct group... but apparently it doesnt in Mac 🤷 
So the issue could be that the reference is lost on Mac or that for some reason this group in Mac has a different Lenght? or is somehow different...

**Comment by DafGreco** [comment]
Hey team! 
This issue is happening again on 0.67

https://github.com/user-attachments/assets/c5b7d743-21f3-468d-8703-6b75bf401e60

**Comment by nearnshaw** [comment]
Hey all, this other ticket was closed bc in theory it was a duplicate with this one

But might not be the case
https://github.com/decentraland/unity-explorer/issues/4180

The other issue specifically refers to audio from a Video Stream form the Scene Admin, not to sounds in general.

**Comment by Kaze-No-Kai** [comment]
Logs from Kim

[Player.log](https://github.com/user-attachments/files/20929350/Player.log)

![Image](https://github.com/user-attachments/assets/0bf77191-17d2-465b-8a25-fc2855521bd1)

**Comment by nicorajcodcl** [comment]
I would like to expreses this issue in simpler terms: 

The Master Volume in the settings doesn't work (at least in Mac).

Could we add a fix for that first? 

I understand there are other (longer) discussions related to adding different sliders for scene audio VS stream audio, etc, in [this ticket 4180](https://github.com/decentraland/unity-explorer/issues/4180). 

Fixing the Master slider is a very userful workaround that we might be able to tackle faster. 

Thanks!

**Comment by anicalbano** [comment]
This issue was reproduced on latest v0.138.0

**Comment by Ludmilafantaniella** [comment]
Couldn't reproduce on Windows or Mac on `v0.147.0`. Tested in Genesis Plaza Thetre (0,7), Casa Roustan (37,-114),  and MusicFestival (-66,-56) and MVFW 2025 (143,-79)— sound settings correctly affect scene audio in all cases. Closing as unnable to repro.

https://github.com/user-attachments/assets/d84b102e-d8bb-417e-81cd-9475d428ca59
